db/views/replying_comments_v01.sql

```diff
@@ -0,0 +1,25 @@
+SELECT
+    read_ribbons.user_id,
+    comments.id as comment_id,
+    read_ribbons.story_id as story_id,
+    comments.parent_comment_id,
+    comments.created_at as comment_created_at,
+    parent_comments.user_id as parent_comment_author_id,
+    comments.user_id as comment_author_id,
+    stories.user_id as story_author_id,
+    (read_ribbons.updated_at < comments.created_at) as is_unread
+FROM
+    read_ribbons
+JOIN
+    comments ON comments.story_id = read_ribbons.story_id
+JOIN
+    stories ON stories.id = comments.story_id
+LEFT JOIN
+    comments parent_comments ON parent_comments.id = comments.parent_comment_id
+WHERE
+    read_ribbons.is_following = 1
+    AND comments.user_id != read_ribbons.user_id
```

Comment:
I slept on this PR and it occurred to me: please add `AND (comments.upvotes - comments.downvotes) < 0 AND (parent_comments.upvotes - parent_comments.downvotes) < 0` (we can memoize a `score` column if this becomes a performance issue). I want this reply tracking because it's going to help sustain really good conversations, I don't want it to sustain flame wars.

Reply:
Added below.